code/modules/vore/appearance/sprite_accessories_taur_vr.dm

```diff
@@ -70,14 +72,17 @@
 	if(isTaurTail(H.tail_style))
 		to_chat(H,"<span class='warning'>Too many legs. TOO MANY LEGS!!</span>")
 		return FALSE
+	if(M.loc != src.loc)
+		if(M.Adjacent(src))
```

Comment:
Why did you omit the user adjacency check?

Reply:
Keep in mind that there are more than two scenarios:
- I buckle someone to me
- I buckle myself to someone

That check is there to make sure that the third scenario is sane:
- I buckle someone to someone else

Without that check, there's nothing to prevent me from buckling someone to someone else across a room because it doesn't care if the user is adjacent, only if M is adjacent to the target.